docker-run

```diff
@@ -68,9 +68,15 @@ def main():
         "--env=USER_ID={}".format(os.getuid() if os.name == "posix" else 1000),
         "--env=GROUP_ID={}".format(os.getgid() if os.name == "posix" else 1000),
         "--env=GIT_BRANCH={}".format(git_branch),
-        "--env=GIT_HASH={}".format(git_hash)
+        "--env=GIT_HASH={}".format(git_hash),
     ])
 
+    if "SSH_AUTH_SOCK" in os.environ:
+        docker_cmd.extend([
+            "--volume={ssh}:{ssh}".format(ssh=os.environ["SSH_AUTH_SOCK"]),
+            "--env=SSH_AUTH_SOCK",
+        ])
```

Comment:
I don't like this. I do not want my ssh private key to be passed to container.
Note that I do not use upgrade script anymore because it run git commands in my place.

Reply:
Do we relay want to be able to start an upgrade on a branch who isn't up do date?
For me not...

I understand why you have many issue with GeoMapFish If you don't want to use it as it's build for for ideological reason ...

Reply:
The first time I've run the upgrade procedure it has remove all my custom scripts without warning (git clean -fd).

Is there some pending pull requests to merge before the upgrade ? Who does upgrade geomapfish ? Who know if we are up to date ? Who is git commands for ? The script or the dev ?

While this script run a git clean -fd in my source folder I won't use it.

If my private ssh key is transfered to the container, not sure I will use it anymore, for security reasons, not for ideology.

Reply:
Note that if ssh key is really needed to clone private repositories in client app projects, it would make sense to put it in client app project docker compose file : 

volumes:
    - $SSH_AUTH_SOCK:$SSH_AUTH_SOCK
  environment:
    - SSH_AUTH_SOCK

Reply:
We are not less secure than before, I don't need that in the docker compose run because I don't use it to run the upgrade process